os_net_config/common.py

```
@@ -508,6 +508,22 @@ def wait_for_vf_driver_binding(pf, vfs, req_driver):
                 )
 
 
+def unset_driverctl_override(pci_address):
+    cmd = ["driverctl", "unset-override", pci_address]
+    try:
+        logger.info("%s: running %s", pci_address, " ".join(cmd))
+        out, err = processutils.execute(*cmd)
+        if err:
+            logger.error("%s: Failed to unbind dpdk interface.", pci_address)
+            return 1
+        return 0
+    except processutils.ProcessExecutionError as exc:
+        logger.error(
+            "%s: Failed to bind interface with dpdk. %s", pci_address, exc
+        )
+        return 1
```

can this be "raise" instead of return ?

---

This would allow the execution to continue and not be stopped due to single failure. This will be part of cleanup routine so it is best that the cleanup is not halted for a single failure.